### PR TITLE
Add global tables blacklist.

### LINKED ISSUE #125: Adding "phinxlog" to blacklist_tables config by default
I don't (and neither does @ADmad) think it makes sense to show the `phinxlog` table used by Phinx - but I'm unsure how we should implement this properly.

It could be implemented as https://gist.github.com/anonymous/c6ac8712102bef1b205f22871cd1cb08, but that's not very customisable - so what do you think should be appropriate? 

https://gist.github.com/anonymous/e090ec5cf5192e08ea7ca3b7cab46359 could easily be done for the default settings, but touching the crud action from within the ViewHelper (https://github.com/FriendsOfCake/crud-view/blob/6e7f359a62887d4aea82b2862e9d3af504184db8/src/Listener/ViewListener.php#L74) seems a bit weird.

Should it be done in https://github.com/FriendsOfCake/crud-view/blob/d8a7b03705275050b59318ea551777b69d527d04/src/Traits/CrudViewConfigTrait.php#L15? That also feels a bit weird to me.

### PR DESCRIPTION
**Pull request by ADmad**
Closes #125

### CODE REVIEW

**Comment by hmic** [comment]
👍

File: config/defaults.php [hunk 1]
```diff
@@ -24,6 +24,9 @@
             ]
         ],
         'timezoneAwareDateTimeWidget' => false,
-        'useAssetCompress' => Plugin::loaded('AssetCompress')
+        'useAssetCompress' => Plugin::loaded('AssetCompress'),
+        'tables_blacklist' => [
```
Comment (jippi) [inline config/defaults.php]: why the change in casing from `tablesBlacklist` to follow the keys above

Comment (hmic) [inline config/defaults.php]: Because this is what the key already looks like in the other places it is used.

Comment (ADmad) [inline config/defaults.php]: All `scaffold.*` action config are underscored but all `Crud.*` global configs are variable cased. I'll change it.

**Comment by hmic** [comment]
would it not, maybe, be better to keep the casing of the key, just to be sure it gets easily found?
maybe introduce
`
scaffold => [
  'tables_blacklist' => ['phinxlog']
]
`
instead? - which would allow to define more global scaffold vars in the course too...

**Comment by josegonzalez** [comment]
Note: won't fix plugins that use phinxlog, and I think we may wish to do that.

**Comment by hmic** [comment]
@josegonzalez can you explain that problem further please?
If a plugin loads it's own configuration (instead) it should/could add the tables_blacklist key itself - and might actually want to add more than just phinxlog to it...
if it does not provide any config, I don't see why this solution would not be working.

**Comment by ADmad** [comment]
@josegonzalez Ping

**Comment by josegonzalez** [comment]
Plugins that use phinx will get separate phinxlog table. Needing to add crud-specific configuration when we could just blacklist them automatically would be sad.

**Comment by ADmad** [comment]
@josegonzalez I am not familiar with how phinxlog tables are used for plugins. Can you make necessary changes to the PR to account for that? Else this can be merged and enhancements done later on.

**Comment by josegonzalez** [comment]
Okay this is fine to merge as is then, just needs docs.

The tables have the `phinxlog` suffix, fyi.
